static/app/views/replays/detail/page.tsx

```diff
@@ -76,7 +79,7 @@ export default function Page({
       <DetailsPageBreadcrumbs orgSlug={orgSlug} replayRecord={replayRecord} />
 
       <ButtonActionsWrapper>
-        <FeedbackButton />
+        {isVideoReplay ? <FeedbackWidgetButton /> : <FeedbackButton />}
```

Comment:
This could render nothing if a user is on self-hosted, feedback is turned off, but video replays are on. Probably ok, but putting it into getsentry could help avoid that.

Reply:
hmm that's a good point. we can probably make a hook in `getsentry` as a followup